test/Microsoft.AspNetCore.Razor.Test.Common/Language/IntegrationTests/IntegrationTestBase.cs

```diff
@@ -227,45 +232,5 @@ protected void AssertSourceMappingsMatchBaseline(RazorCodeDocument document)
 
             Assert.Equal(baseline, actual);
         }
-
-        protected class ApiSetsIRTestAdapter : IntermediateNodePassBase, IRazorOptimizationPass
```

Comment:
Doing this in an optimization pass is the wrong thing. I reworked this feature to use the default document classifier, which is how we'd tell users to do it.

Reply:
Also I had to do something here, because the `MetadataAttributePass` was seeing the uninitialized values.